README.md

```diff
@@ -1,8 +1,6 @@
-# curriculum-databases-projects-template
-
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
+# Vet Clinic Database
 
```

Comment:
- [ ] Kindly include a description of this project to make your README more informative and descriptive.

Reply:
done